frontend/package.json

```diff
@@ -24,7 +24,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "npx prettier --write ./src/**/*.{js,jsx,ts,tsx} && npx eslint --fix ./src/**/*.{js,jsx,ts,tsx}"
```

Comment:
I don't think prettier can ever throw an error, so it doesn't need to be a part of CI

Reply:
Good catch!

I've changed the `--write` flag in the prettier command to `--check`. This should run and fail if it finds stuff that prettier could format.

This way we'd make sure that anything committed to the repo should have had prettier run on it first. (And if people have the pre-commit hook set up, that should be done automatically for them)